app/assets/stylesheets/components/c-base.scss

```diff
@@ -0,0 +1,13 @@
+.c-main__headers {
```

Comment:
base.scss should just contain base styles on base elements, i.e. `body` etc.  This should be in the `c-main` file